index.html

```diff
@@ -0,0 +1,611 @@
+<!doctype html>
+<html class="no-js" lang="">
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="x-ua-compatible" content="ie=edge">
+    <title>AppBox | Build Beautiful Mobile Apps</title>
+    <meta name="description" content="Best way to build and present your mobile app">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link href='https://fonts.googleapis.com/css?family=Roboto:400,400italic,300,500|Raleway:600,500,400,300|Roboto+Condensed:400,700' rel='stylesheet' type='text/css'>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/Swiper/3.1.2/css/swiper.min.css">
+    <link rel="stylesheet" href="css/main.css">
+  </head>
+  <body>
+    <header id="top">
+      <h1 class="logo">AppBox</h1>
+      <a id="menuToggle" class="menu-toggle" href="#nav">MENU</a>
+      <nav role="navigation">
+        <ul id="nav" class="menu">
+          <li><a class="nav-link" href="#features">Features</a></li>
+          <li><a class="nav-link" href="#why">Why AppBox</a></li>
+          <li><a class="nav-link" href="#screens">Screenshots</a></li>
+          <li><a class="nav-link" href="#more">More Apps</a></li>
+          <li><a class="nav-link" href="#pricing">Pricing</a></li>
+          <li><a class="nav-link" href="#featured">Featured</a></li>
+          <li><a class="nav-link" href="#support">Support</a></li>
+          <li>
+            <div class="nav-link">
+              <form class="search-field">
+                <label class="" for="search"></label>
+                <input id="search" aria-label="search field" type="search">
+              </form>
+            </div>
+          </li>
+          <li class="close-nav"><a href="#menuToggle"></a></li>
+        </ul>
+      </nav>
+    </header>
+    <main id="content">
+      <section class="hero-section" id="backtotop">
+        <div data-div="best-way">
+          <div class="hero-text-wrapper">
+            <h1>Best Way To Present Your Beautiful Mobile App</h1>
+            <p>AppBox is full of awesome features. Its designed for<br> <strong>App Landing and App Store site</strong></p>
+            <figure>
+              <video src="#" width="150" height="85" poster="/img/video-placeholder.jpg"></video>
```

Comment:
This `<video>` element is incomplete, it should have either a `src` attribute with a URL for a video file, or one or more child `<source>` elements. 

Also, `<video>` elements, just like `<img>` elements, can and should omit the `width` and `height` attributes in favor of sizing with CSS.